src/cpp/fastdds/publisher/Publisher.cpp

```diff
@@ -61,7 +61,9 @@ ReturnCode_t Publisher::enable()
     }
 
     enable_ = true;
```

Comment:
This is not needed anymore

Reply:
Yes, it is. Inside `impl_->enable()`, we may call the `enable()` method on previously created writers, which will fail if the publisher is not enabled.